android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.kt

```diff
@@ -110,6 +111,16 @@ class ReactExoplayerViewManager(private val config: ReactExoplayerConfig) : View
         videoView.setAdTagUrl(adTagUrl)
     }
 
+    @ReactProp(name = PROP_AD_LANGUAGE)
+    fun setAdLanguage(videoView: ReactExoplayerView, languageString: String?) {
+        if (TextUtils.isEmpty(languageString)) {
+            videoView.setAdLanguage(null) // Maybe "en" default?
```

Comment:
I think we should leave default behavior when language is non passed, so i suggest to remove this line

Reply:
null will fallback on default behavior, so ok for me